### PR TITLE
Don't require act_as_admin permssion

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Followup on https://github.com/mattermost/mattermost-plugin-apps/pull/255

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-38243

### CODE REVIEW

File: src/manifest.json [hunk 1]
```diff
@@ -33,7 +32,6 @@
         "expand": {
             "acting_user": "all",
             "channel": "summary",
-            "admin_access_token": "all",
```
Comment (hanzei) [inline src/manifest.json:36]: The token doesn't seem to be used: https://github.com/mattermost/mattermost-app-zendesk/blob/5c2d8805564e71480da21594e4b811c4d4f42cf0/src/bindings/index.ts#L18-L18

File: src/manifest.json [hunk 2]
```diff
@@ -33,7 +32,6 @@
         "expand": {
             "acting_user": "all",
             "channel": "summary",
```
Comment (hanzei) [inline src/manifest.json:34]: Is that needed?

File: src/forms/create_ticket.ts [hunk 3]
```diff
@@ -34,7 +34,7 @@ export async function newCreateTicketForm(call: AppCallRequest): Promise<AppForm
         botAccessToken: context.bot_access_token,
         adminAccessToken: context.admin_access_token,
     };
-    const mmClient = newMMClient(mmOptions).asAdmin();
+    const mmClient = newMMClient(mmOptions).asActingUser();
```
Comment (hanzei) [inline src/forms/create_ticket.ts:37]: The token is already getting expanded in https://github.com/mattermost/mattermost-app-zendesk/blob/3ed148a73f308404cc52709d4d1cb842f3eba4ab/src/bindings/bindings.ts#L146-L146

**Review by larkox — APPROVED** [review]

**Comment by hanzei** [comment]
@jfrerich I'm incident to keep the `asAdmin` methods as this is the only JS/TS app right now and the prime example. Other developers looking at the code might use it as a starting point.

**Comment by hanzei** [comment]
Let's test this as part of release testing.

**Comment by hanzei** [comment]
@jfrerich I went ahead and removed all reference to admin_access_token as https://github.com/mattermost/mattermost-plugin-apps/pull/284 will remove it.

**Pull request merged**